packages/experimental/Dropdown/src/Dropdown/Dropdown.tsx

```diff
@@ -1,16 +1,49 @@
-import { compose, UseSlots } from '@fluentui-react-native/framework';
+/** @jsx withSlots */
+import { IViewProps } from '@fluentui-react-native/adapters';
+import { ButtonV1 as Button, ButtonProps } from '@fluentui-react-native/button';
+import { buildUseTokens, compressible, useSlot, UseTokens, withSlots } from '@fluentui-react-native/framework';
 import React from 'react';
 import { View } from 'react-native';
-import { dropdownName, DropdownProps, DropdownType } from './Dropdown.types';
+import { Path, Svg, SvgProps } from 'react-native-svg';
+import { dropdownName, DropdownProps, DropdownTokens } from './Dropdown.types';
 
-export const Dropdown = compose<DropdownType>({
-  displayName: dropdownName,
-  slots: {
-    root: View,
-  },
-  useRender: (_userProps: DropdownProps, _useSLots: UseSlots<DropdownType>) => {
-    return (_final: DropdownProps, ..._children: React.ReactNode[]) => {
-      return null;
-    };
-  },
-});
+const Dropdown = compressible<DropdownProps, DropdownTokens>((props: DropdownProps, _useTokens: UseTokens<DropdownTokens>) => {
+  const onButtonClick = React.useCallback(() => {}, []); //eslint-disable-line
+  const buttonProps: ButtonProps = React.useMemo(
+    () => ({
+      onClick: onButtonClick,
+    }),
+    [onButtonClick],
+  );
+
+  const expandIconProps: SvgProps = React.useMemo(
```

Comment:
This could be a const object outside of the component, but it will probably take tokens in the future so I think it makes sense to keep it as a memoized function which returns a constant